index.html

```diff
@@ -9,6 +9,29 @@
         <link rel="stylesheet" href="main.css">
     </head>
     <body>
-        
+        <nav>
+            <p>Welcome</p>
+            <a href="#"><img src="img/Icon-Menu.svg" alt="Menu"></a>
```

Comment:
- [ ] Please consider wrapping the icon menu with a `nav` tag as the icon menu which is also known as the hamburger menu consists of navigation links and also to improve the semantic structure of your HTML😄. You can read more on [SEMANTIC HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). See my suggestions below.

```suggestion
          <nav>
            <a href="#"><img src="img/Icon-Menu.svg" alt="Menu"></a>
          </nav>
```